### PR TITLE
Add support for django 1.11

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

File: django_tables/base.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import copy
+from collections import OrderedDict
+
 from django.http import Http404
 from django.core import paginator
-from django.utils.datastructures import SortedDict
```
Comment (jlward) [inline django_tables/base.py:4]: `SortedDict` -> `OrderedDict`

File: django_tables/base.py [hunk 2]
```diff
@@ -147,9 +149,9 @@ def polarize(self, reverse, names=()):
             order_by_tuple = [
                 (
                     # add either untouched, or reversed
-                    (names and rmprefix(o) not in names)
-                    and [o]
-                    or [prefix+rmprefix(o)]
+                    (names and rmprefix(o) not in names) and
```
Comment (jlward) [inline django_tables/base.py:152]: Some minor flake8

File: django_tables/tests/testapp/models.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from django.db import models
```
Comment (jlward) [inline django_tables/tests/testapp/models.py:1]: This was pulled out of test_models.py. We were doing terrible, terrible things.

File: run_tests.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```
Comment (jlward) [inline run_tests.py:1]: This file is new. It's the pattern I've been following for testing our third party apps that have not aged well.

File: tox.ini [hunk 5]
```diff
@@ -5,15 +5,13 @@
...
     # py34-django{17,18}
 
 [testenv]
-commands = nosetests
+commands = {envpython} run_tests.py
 deps =
     nose
-    django14: Django>=1.4, <1.5
-    django16: Django>=1.6, <1.7
-    django17: Django>=1.7, <1.8
```
Comment (bocribbz) [inline tox.ini:18]: are the previous django version failing to pass the tests? or you just dont want to have to bother with them?

Comment (jlward) [reply tox.ini:18]: I don't want to bother with them. They are not supported any longer anyway.

File: tests/testapp/models.py [hunk 6]
```diff
@@ -1 +0,0 @@
-"""Empty demo app our tests can assign models to."""
```
Comment (bocribbz) [inline tests/testapp/models.py:1]: is this file still in the repo? its not very clear from the diff.

Comment (jlward) [reply tests/testapp/models.py:1]: I moved it to `django_tables/tests/testapp/models.py` and this docstring eventually got removed.

I actually moved `tests` into `django_tables` which I was hoping wouldn't mess with the diff too much.

**Comment by bocribbz** [comment]
Looks great to me!

**Review by bocribbz — APPROVED** [review]

**Pull request merged**